packages/salesforcedx-utils-vscode/src/services/telemetry.ts

```diff
@@ -231,18 +281,34 @@ export class TelemetryService {
             aggregatedMeasurements.executionTime = this.getEndHRTime(hrstart);
           }
         }
-        this.reporter!.sendTelemetryEvent(
-          'commandExecution',
-          aggregatedProps,
-          aggregatedMeasurements
-        );
+        this.reporters.forEach(reporter => {
+          reporter.sendTelemetryEvent(
+            'commandExecution',
+            aggregatedProps,
+            aggregatedMeasurements
+          );
+        });
       }
     });
   }
 
   public sendException(name: string, message: string) {
     this.validateTelemetry(() => {
-      this.reporter!.sendExceptionEvent(name, message);
+      this.reporters.forEach(reporter => {
+        try {
+          reporter.sendExceptionEvent(name, message);
+        } catch (error) {
+          console.log(
```

Comment:
The t/c is here in the service to catch in case any of the reporters has a problem trying to record telem on the exception.  This t/c is inside the for-each so it should already be catching in this scenario for all reporters here.